src/main.js

```diff
@@ -102,28 +106,40 @@ var savedPosters = [];
 var currentPoster;
 
 // event listeners go here 👇
+window.addEventListener("load", makeNewPoster);
+showRandomButton.addEventListener('click', makeNewPoster);
 
 // functions and event handlers go here 👇
 // (we've provided one for you to get you started)!
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
 
-//do we need to link this or can we copy into main.js?
```

Comment:
We should probably delete this.

Reply:
Yes, agreed we will need to refactor code.